src/hackstead/mod.rs

```diff
@@ -1,27 +1,55 @@
-use actix_web::{get, web, HttpResponse};
+use crate::data;
+use actix_web::{get, post, web, HttpResponse};
+use log::*;
 use hcor::{Hackstead, ServiceError, UserContact};
 
+#[cfg(test)]
+mod test;
+
 #[get("/hackstead/")]
 pub async fn get_hackstead(form: web::Json<UserContact>) -> Result<HttpResponse, ServiceError> {
-    log::debug!("servicing get_hackstead request");
```

Comment:
This makes sense to keep in as a debug log

Reply:
yeah, you're probably right about that. I removed it remembering incorrectly that async-web would do a better job of logging when the route was called automatically.